app/models/wishlist.rb

```diff
@@ -0,0 +1,4 @@
+class Wishlist < ApplicationRecord
+  validates :name, length: { in: 3..30 }
+  validates :description, length: { in: 10..150 }
```

Comment:
I agree we shouldn't require any description. But I think setting the max length could be good to not let someone accidentally paste something way to big. Not sure what the exact limit should be, maybe something like 1000 chars?